tests/python_client/milvus_client/test_milvus_client_struct_array.py

```diff
@@ -0,0 +1,4054 @@
+        struct_schema.add_field("clip_embedding2", DataType.FLOAT_VECTOR, dim=dim)
+
+        # Add struct array field
+        schema.add_field(
+            "clips",
+            datatype=DataType.ARRAY,
+            element_type=DataType.STRUCT,
+            struct_schema=struct_schema,
+            max_capacity=capacity,
+        )
+
+        return schema
+
+    @pytest.mark.tags(CaseLabel.L0)
+    def test_create_struct_array_with_clip_embedding1(self):
+        """
+        target: test create collection with struct array containing vector field
+        method: create collection with struct array field that contains FLOAT_VECTOR
+        expected: creation successful
+        """
+        collection_name = cf.gen_unique_str(f"{prefix}_basic")
+
+        client = self._client()
+
+        # Create schema with struct array containing vector field
+        schema = self.create_struct_array_schema(client, dim=default_dim)
+
+        # Create collection
+        collection_name = cf.gen_unique_str(f"{prefix}_vector")
+        res, check = self.create_collection(client, collection_name, schema=schema)
+        assert check
+
+        # Verify collection exists
+        has_collection, _ = self.has_collection(client, collection_name)
+        assert has_collection
+
+    @pytest.mark.tags(CaseLabel.L0)
+    def test_create_struct_array_with_scalar_fields(self):
+        """
+        target: test create collection with struct array containing only scalar fields
+        method: create collection with struct array field that contains only scalar types
+        expected: creation successful
+        """
+        collection_name = cf.gen_unique_str(f"{prefix}_basic")
+
+        client = self._client()
+
+        # Create schema
+        schema = client.create_schema(auto_id=False, enable_dynamic_field=False)
+        schema.add_field(field_name="id", datatype=DataType.INT64, is_primary=True)
+        schema.add_field(
+            field_name="normal_vector", datatype=DataType.FLOAT_VECTOR, dim=default_dim
+        )
+
+        # Create struct schema with only scalar fields
+        struct_schema = client.create_struct_field_schema()
+        struct_schema.add_field("int_field", DataType.INT64)
+        struct_schema.add_field("float_field", DataType.FLOAT)
+        struct_schema.add_field("string_field", DataType.VARCHAR, max_length=512)
+        struct_schema.add_field("bool_field", DataType.BOOL)
+
+        # Add struct array field
+        schema.add_field(
+            "scalar_struct",
+            datatype=DataType.ARRAY,
+            element_type=DataType.STRUCT,
+            struct_schema=struct_schema,
+            max_capacity=50,
+        )
+
+        # Create collection
+        collection_name = cf.gen_unique_str(f"{prefix}_scalar")
+        res, check = self.create_collection(client, collection_name, schema=schema)
+        assert check
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.parametrize(
```

Comment:
suggest update this case to L2, as in should be covered in the test_create_struct_with_various_scalar_types